### ISSUE TITLE
[request] SOTA EfficientNet

### ISSUE THREAD

**Issue by Kulikovpavel** [Cadene/pretrained-models.pytorch#158]
Code and pretrained up to B3
https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet

**Comment by rwightman** [comment]
Remi is welcome to port into his style: https://github.com/rwightman/pytorch-image-models/blob/master/models/gen_efficientnet.py

**Comment by rwightman** [comment]
Needs a padding hack to work well with the TF weights though

**Comment by finlay-liu** [comment]
This repo:
https://github.com/lukemelas/EfficientNet-PyTorch#loading-pretrained-models
